data.sql

```diff
@@ -33,3 +33,128 @@ UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Bob') WH
 UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Melody Pond') WHERE name IN ('Charmandor', 'Squirtle', 'Blossom');
 UPDATE animals SELECT owner_id = (SELECT id from owners WHERE full_name = 'Dean Winchester') WHERE name IN ('Angemon', 'Boarmon');
 
+
+ INSERT INTO vets(name, age, date_of_graduation) VALUES
+  ('William Tatcher', 45, '2000-04-23'),
+  ('Maisy Smith', 26, '2019-01-17'),
+  ('Stephanie Mendez', 64, '1981-05-04'),
+  ('Jack Harkness', 38, '2008-06-08');
+
+
+INSERT INTO  specialization(vet_id, species_id) VALUES(
+  (SELECT id FROM vets WHERE full_name = 'William Tatcher'),
+  (SELECT id FROM species WHERE name = 'Pokemon')
+ );
+
+INSERT INTO specialization(vet_id, species_id) VALUES
+(
+  (SELECT id FROM vets WHERE name = 'Stephanie Mendez'),
+  (SELECT id FROM species WHERE name = 'Digimon')
+),
+(
+  (SELECT id FROM vets WHERE name = 'Stephanie Mendez'),
+  (SELECT id from species WHERE name = 'Pokemon')
+);
+
```

Comment:
![image](https://github.com/batoolfatima2135/Vet-Clinic/assets/93039359/9db761fb-313d-4304-ae7b-3737d1204c23)

- [ ] Great so far but it would be nice if you add the missing query in your data.sql file to make your data up to project requirements 👍

Reply:
My pleasure 😊